package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "biscoint-api-node",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
versions in `package-lock.json` and `package.json` are different

Reply:
ok

Reply:
solved